distribution/docs/src/main/resources/_contents/configuring-from-import-contents.adoc

```diff
@@ -7,8 +7,8 @@ The Configuration Export/Import capability allows administrators to export the c
 * Importing configuration files is only guaranteed to work when importing files from the same ${branding} version.
   Importing from a different version is not recommended as it may cause the new ${branding} instance to be incorrectly configured and become unusable.
 * All configurations editable in the Admin Console will be exported to `<${branding}_HOME>/etc/exported/etc`.
-* All other configuration files (system configurations) will be put under `etc/exported` followed by their relative path from `${branding}_HOME`.
-  For instance, if a key store file was located under `<${branding}_HOME>/keystores/keystore.jks`, it will be exported to `<${branding}_HOME>/etc/exported/keystore/keystore.jks`.
+* All other configuration files (system configurations) will be put under `etc/exported/${id}` followed by their relative path from `${branding}_HOME`.
```

Comment:
Should we be mentioning etc/exported in the docs?  An admin can enter any directory he/she wants for export.